documentation/4_Perspectives_of_the_Reference_Architecture_Model/4_2_Certification_Perspective/RulesOfProcedure/rules-of-procedure.md

```diff
@@ -0,0 +1,430 @@
+
+## 2. Certification body and evaluation facility    
+### §2.1 Certification body      
+1. The IDS certification is carried out by an independent and technically suitable certification body.
+The certification body shall carry out its activities in a non-discriminatory and impartial manner.
+2. The certification body may be a legal entity or a separate part of a legal entity.
+3. The certification body must have the financial stability and resources necessary for the activity.
+4. The certification body shall ensure the confidentiality of information on and arising from the certification procedures.
+5. In the event of a positive evaluation, the certification body issues a certificate to the applicant.
+
+### § 2.2 Certifier
+1. Certifiers are natural persons who carry out the assessment of the evaluation for a certification body.
+The activity of the certifier may be carried out through the collaboration of several natural persons.
+In this case it is sufficient if the technical requirements according to paragraphs 2 to 5 are fulfilled by the persons concerned as a group.
+The personal requirements according to paragraphs 4 and 7 must be fulfilled by each person.
+2. Certifiers shall demonstrate sufficient professional competence for the assessments they carry out.
+3. A university degree or equivalent qualification is required for the certification activity.
+4. The certifier must have sufficient knowledge in the areas relevant for certification.
+In particular, the certifier shall have sufficient knowledge for certification in the following areas:
+  * a. technical requirements for data security;
+  * b. ISO/IEC 270xy standards (or equivalent);
+  * c. ISO/IEC 15408 standard (or equivalent);
+  * d. IDS certification procedure;
+  * e. IDSA reference architecture.
+5. Sufficient experience for certification requires at least four years (full-time equivalent) experience in the field of information technology and at least two years (full-time equivalent) experience as a certifier or at least four years (full-time equivalent) experience as an evaluator of products and/or organizations.
+6. Certifiers must have the reliability required to perform their duties properly based on their personal characteristics, behavior, and skills.
+7. Certifiers must be independent within the meaning of § 2.7 paragraph 3.
+They shall carry out their duties in a non-discriminatory and impartial manner.
+8. The certification body shall ensure that the certifiers it employs meet the requirements set out in paragraphs 2 to 7.
+
+### §2.3 Evaluation facility      
+1. Independent and technically suitable evaluation facilities can be commissioned to evaluate the operational environment of a participating organization or the components of a manufacturer in order to issue an IDS certificate.
```

Comment:
```suggestion
1. Independent and technically suitable evaluation facilities can be commissioned by the applicant to evaluate the operational environment of a participating organization or the components of a manufacturer in order to receive an IDS certificate from the IDSA certification body.
```

Reply:
The old wording sounded like the EF issues the certificate.

Reply:
I agree and integrate the clarification